src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_IndexerAccess.cs

```diff
@@ -53,7 +54,7 @@ private BoundExpression MakeDynamicGetIndex(
             ImmutableArray<string?> argumentNames,
             ImmutableArray<RefKind> refKinds)
         {
-            // If we are calling a method on a NoPIA type, we need to embed all methods/properties
+            // If we are calling a method on a NoPIA typeWithElementType, we need to embed all methods/properties
```

Comment:
Most of these comment changes don't make sense?

Reply:
I think they're unintended. I'll revert them.